.github/workflows/publish-operator-bundle.yaml

```diff
@@ -0,0 +1,74 @@
+name: "Publish operator bundle"
+
+on:
+  push:
+    paths:
+      - '.github/workflows/publish-operator-bundle.yaml'
+      - 'bundle.Dockerfile'
+    branches:
+      - main
+    tags:
+      - 'v*'
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-20.04
```

Comment:
I'd go with the latest LTS version, `ubuntu-22.04`

Reply:
I copied the value from the other workflows but totally agree!